<!--xml-->
<launch>
 <node name="$(env ROS_HOSTNAME)_timeslave_node" pkg="icarus_rover_v2" type="timeslave_node"  output="screen" clear_params="true">
        <param name="startup_delay"       value="20.0"/>
	    <param name="verbosity_level"       value="NOTICE"/> 
        <param name="require_pps_to_start"  value="false"/>
        <param name="loop1_rate"            value="10.0"/> <!-- publish -->
</node>
</launch>
